java/example/common/src/main/java/org/eclipse/keyple/example/common/calypso/pc/transaction/CalypsoUtilities.java

```diff
@@ -89,14 +92,39 @@ public static SamResource getDefaultSamResource() throws KeypleException {
          *
          * (We expect the right is inserted)
          */
-        SamResource samResource = checkSamAndOpenChannel(samReader);
-
-        return samResource;
+        return checkSamAndOpenChannel(samReader);
     }
 
-    public static SecuritySettings getSecuritySettings() {
+    public static PoSecuritySettings getSecuritySettings() {
```

Comment:
This helper method should maybe localized outside of the Keyple Calypso library, maybe in org.eclipse.keyple.example.common.calypso.pc.transaction.CalypsoUtilities